geoportal/c2cgeoportal_geoportal/views/memory.py

```diff
@@ -28,65 +28,79 @@
 # either expressed or implied, of the FreeBSD Project.
 
 
-import inspect
 import logging
-import sys
+from typing import Dict, List, Any
 
+from c2cwsgiutils import broadcast
 from pyramid.view import view_config
 
 from c2cgeoportal_geoportal.views import entry, raster
-from c2cwsgiutils import broadcast
+from pympler import asizeof, muppy
 
-log = logging.getLogger(__name__)
+LOG = logging.getLogger(__name__)
 
 
 @view_config(route_name="memory", renderer="fastjson")
 def memory(request):
-    del request
-    return _memory()
+    return _memory(
```

Comment:
I doubt this code has been tested. broadcast works only with named parameters...

Reply:
:-), thanks